shared/fs/index.js

```diff
@@ -102,6 +98,28 @@ class Files extends React.PureComponent<FolderProps> {
   }
 }
 
+const folderBoxStyle = {...globalStyles.flexBoxColumn, flex: 1, justifyContent: 'space-between'}
```

Comment:
Are we having a convention here to always have style definitions after the components?